### PR TITLE
added not about offline installation of the apps

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #4373

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-smartscreen.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ manager: dansimp
 <!--Description-->
 Added in Windows 10, version 1703. Allows IT Admins to control whether users are allowed to install apps from places other than the Store.
 
+> [!Note]
+> This policy will block installation only while the device is online. To block offline installation too, policies **SmartScreen/PreventOverrideForFilesInShell** and **SmartScreen/EnableSmartScreenInShell**  should also be enabled.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-smartscreen.md]:
```suggestion
> This policy will block installation only while the device is online. To block offline installation too, **SmartScreen/PreventOverrideForFilesInShell** and **SmartScreen/EnableSmartScreenInShell**  policies  should also be enabled.
```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were applied,
Thank you!

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #4373 

Thank you.

**Pull request merged**